src/main/resources/data/abnormals_delight/recipes/atmospheric/cutting/gilia.json

```diff
@@ -0,0 +1,17 @@
+{
+  "type": "farmersdelight:cutting",
+  "ingredients": [
+    {
+      "item": "atmospheric:gilia"
+    }
+  ],
+  "tool": {
+    "tag": "forge:tools/knives"
+  },
+  "result": [
+    {
+      "item": "minecraft:purple_dye",
+      "count": 2
+    }
+  ]

```

Comment:
All recipes will need a condition segment below the result, like so:
```
  "conditions": [
    {
      "type": "forge:mod_loaded",
      "modid": "atmospheric"
    }
  ]
```

Reply:
Should be good now :thumbsup: